adapter/repository/in-memory-simple/src/main/resources/gtfs_spec.asciipb

```diff
@@ -1311,7 +1311,7 @@ csvspec: {
     value_required: true
     type: { type: INT_DEC }
     intmin: 0
-    intmax: 300000
+    intmax: 5500000
```

Comment:
Why is this `5500000`?

Reply:
After a few iteration I realized that the previous value was too small. I set it to 5500000 in order to ensure that the program run correctly and does not generate too many out of range notice. 

This seems to be a calibration issue, that could require to explore a large quantity of GTFS datasets. Maybe we should to open an issue on this matter as I realize that we need to go further on this?

Reply:
Is there a downside to setting this to the max int value for 32 bit, which would be `2147483647`?

That should handle any reasonable value.

Reply:
We can't see any downside. Will modify.